Apps/Common/src/AccessManagement/Administration/KeycloakUserAdminDelegate.cs

```diff
@@ -85,6 +99,51 @@ public UserRepresentation GetUser(Guid userId, JWTModel jwtModel)
             throw new NotImplementedException();
         }
 
+        /// <inheritdoc/>
+        public async Task<RequestResult<IEnumerable<UserRepresentation>>> GetUsers(IdentityAccessRole role, JWTModel jwtModel)
+        {
+            using Activity? activity = Source.StartActivity();
+
+            this.logger.LogDebug("Getting users for role: {Role} ...", role);
+
+            RequestResult<IEnumerable<UserRepresentation>> retVal = new()
+            {
+                ResultStatus = ResultType.Error,
+            };
+
+            Uri baseUri = new Uri(this.configuration.GetSection(KEYCLOAKADMIN).GetValue<string>(GetRolesUrlKey));
+            HttpClient client = this.CreateHttpClient(baseUri, jwtModel.AccessToken);
+            string uri = $"{role}/users";
+
+            HttpResponseMessage response = await client.GetAsync(new Uri(uri, UriKind.Relative)).ConfigureAwait(true);
+            string payload = await response.Content.ReadAsStringAsync().ConfigureAwait(true);
+            this.logger.LogDebug("Getting users for role: {Role}, response payload: {Payload} ...", role, payload);
+
+            if (response.IsSuccessStatusCode)
+            {
+                retVal.ResultStatus = ResultType.Success;
+
+                List<UserRepresentation>? users = JsonSerializer.Deserialize<List<UserRepresentation>>(payload);
+                if (users != null)
+                {
+                    retVal.ResourcePayload = users.ToList();
+                    retVal.TotalResultCount = users.ToList().Count;
+                }
+            }
+            else
+            {
+                this.logger.LogError("Unable to connect to endpoint: {Endpoint}, HTTP Error: {Response} ...", baseUri + uri, response.StatusCode);
+
+                retVal.ResultError = new RequestResultError()
+                {
+                    ResultMessage = $"Unable to connect to endpoint: {baseUri}{uri}, HTTP Error: {response.StatusCode}.",
```

Comment:
Why not generate this string once and then you can log the same string on 135 above

Reply:
I wanted to use a message defined in a string for both ResultError and Logger but I got a warning from the logger.  The logger gave a warning when the message was created before hand.